code/__DEFINES/misc.dm

```diff
@@ -253,7 +253,8 @@
 #define MATRIX_DEFAULT list(1, 0, 0, 0,\
                             0, 1, 0, 0,\
                             0, 0, 1, 0,\
-                            0, 0, 0, 1)
+                            0, 0, 0, 1,\
+                            0, 0, 0, 0)
```

Comment:
Yeah, that's what I figured when I went back through it. It's much more bare-bones now and difflist isn't a part of it anymore